src/main/java/mate/academy/mybookshop/annotation/PasswordMatchesValidator.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.mybookshop.annotation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.mybookshop.dto.UserRegistrationRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PasswordMatchesValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String firstFieldName;
+    private String secondFieldName;
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        firstFieldName = constraintAnnotation.first();
+        secondFieldName = constraintAnnotation.second();
+    }
```

Comment:
What is the reason for secondFieldName, firstFieldName? This fields is not used

Reply:
thx